### PR TITLE
Project list improvements.

### PR DESCRIPTION
**Pull request by monday8am**
- New colours on projects. (to check with team, like iOS version) @masterrr @taavirehemagi 
- Projects organised by workspace as previous versions. Client info available.
- New project creation jumps directly to EditTimeEntry screen.
- RecyclerView support.
- Minor improvements on usability.

### CODE REVIEW

**Comment by taavirehemagi** [comment]
Looks good to me!

File: Joey/UI/Adapters/ProjectListAdapter.cs [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+            public FrameLayout TasksFrameLayout { get; private set; }
+
+            public TextView TasksTextView { get; private set; }
+
+            public ImageView TasksImageView { get; private set; }
+
+            public ProjectListItemHolder (ProjectListAdapter adapter, View root) : base (root)
+            {
+                this.adapter = adapter;
+                ColorView = root; //root.FindViewById<View> (Resource.Id.ColorView);
```
Comment (masterrr) [inline Joey/UI/Adapters/ProjectListAdapter.cs]: comment not needed? :stuck_out_tongue:

File: Joey/UI/Adapters/ProjectListAdapter.cs [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+            private readonly ProjectListAdapter adapter;
+
+            public View ColorView { get; private set; }
+
+            public TextView ProjectTextView { get; private set; }
+
+            public NoProjectListItemHolder (ProjectListAdapter adapter, View root) : base (root)
+            {
+                this.adapter = adapter;
+                ColorView = root; //root.FindViewById<View> (Resource.Id.ColorView);
```
Comment (masterrr) [inline Joey/UI/Adapters/ProjectListAdapter.cs]: same comment

File: Joey/UI/Adapters/ProjectListAdapter.cs [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+
+        private class NewProjectListItemHolder : RecycledBindableViewHolder<ProjectListView.Project>, View.IOnClickListener
+        {
+            private readonly ProjectListAdapter adapter;
+
+            public TextView ProjectTextView { get; private set; }
+
+            public NewProjectListItemHolder (ProjectListAdapter adapter, View root) : base (root)
+            {
+                //ColorView = root.FindViewById<View> (Resource.Id.ColorView);
```
Comment (masterrr) [inline Joey/UI/Adapters/ProjectListAdapter.cs]: comment

File: Joey/UI/Adapters/ProjectListAdapter.cs (hunk 2 above)
Comment (monday8am) [inline Joey/UI/Adapters/ProjectListAdapter.cs]: @masterrr I'm waiting for your opinion about the colours:

- The colours works ok in iOS.
- Without colours the Project Item looks very similar to TimeEntry item. 

Yes yes ( don't need to repeat three times :P), I'll remove the comments with the colour issue defined.

Comment (masterrr) [inline Joey/UI/Adapters/ProjectListAdapter.cs]: @monday8am what opinion do you want? Just compiled your branch, don't get your idea :)

Comment (monday8am) [inline Joey/UI/Adapters/ProjectListAdapter.cs]: Did you check the project list? If not, please check.

**Pull request merged**